tasks/create-promotional-video-about-the-Labyrinth-game.md

```diff
@@ -0,0 +1,38 @@
+
+
+---
+
+# Labyrinth: Create a Promotional Video about the Labyrinth Game
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth.
+To promote the labyrinth, create a promotional video.
+
+REQUIREMENTS:
+
+- Explore [Video Creating](https://en.wikipedia.org/wiki/List_of_video_editing_software#Free_and_open-source)
+- Include the FOSSASIA logo. You find **FOSSASIA Artwork** on this repository: ***https://github.com/fossasia/fossasia-artwork***
+- Use existing [YouTube videos from the FOSSASIA Channel](https://youtube.com/fossasiaorg) and remix them
+- The video should be uploaded publicly on the web domain and if possible shown in your school.
+- Link the video on the program website. Add it to ***https://github.com/fossasia/gci17.fossasia.org***
+- Tweet the video on Twitter, Weibo or other tweet like channel and include the video in the tweet itself. For example something like "Check out this video about #FOSSASIA http-youtube-link @hpdang @mariobehling @fossasia @catallman More at https://github.com/fossasia/labyrinth"
+- Post about your video on Facebook and other social media channels. Your post should include the link to the video.
+
+EXPECTED OUTCOME:
+
+- Video made using Open Source on YouTube using FOSSASIA Artwork/logo
+- Public screening
+- Link on FOSSASIA Code-In site
+- Tweet and post it on social media
```

Comment:
Should we add it as a link to the labyrinth website?

Reply:
We can upload the video to youtube and then can share the links.

> Any suggestions ?